chain/epoch-manager/src/shard_tracker.rs

```diff
@@ -90,6 +96,9 @@ impl ShardTracker {
                 let subset = &schedule[index as usize];
                 Ok(subset.contains(&shard_id))
             }
+            TrackedConfig::ShadowValidator(account_id) => {
+                self.epoch_manager.cares_about_shard_in_epoch(*epoch_id, account_id, shard_id)
```

Comment:
can we also give some warning somewhere (potentially here) if the accountid is not eligible to be a chunk validator as all? I was thinking about when loading the config but at that point I am not sure if the node will have the full info about the other validators.

Reply:
Will look into it in a separate PR, together with nayduck test.